ci/experimental/build_docker.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -e
+#
+# Build Snowflake Python Connector in Docker
+# NOTES:
+#   - To compile only a specific version(s) pass in versions like: `./build_docker.sh "3.5 3.6"`
+set -o pipefail
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source $THIS_DIR/set_base_image.sh
+CONNECTOR_DIR="$(dirname "$( dirname "${THIS_DIR}")")"
+echo "conndir= $CONNECTOR_DIR, thisdir=$THIS_DIR"
+mkdir -p $CONNECTOR_DIR/dist
+cd $THIS_DIR/docker/connector_build
+
+CONTAINER_NAME=build_pyconnector
```

Comment:
Wouldn't this clash with the other non aarch docker image?
```suggestion
CONTAINER_NAME="build_pyconnector_aarch"
```